src/org/labkey/test/tests/list/ListDateAndTimeTest.java

```diff
@@ -1039,7 +1039,8 @@ public void testInvalidDateAndTimeInsert() throws IOException, CommandException
 
         log("Validate adding entries in bulk will give a meaningful error with a bad format.");
 
-        String expectedErrorMsgFormat = "Could not convert value '%s' (String) for %s field '%s'";
+        String expectedErrorMsgFormat = "'%s’ is not a valid %s for %s using U.S. date parsing (MDY)";
+        String expectedTimeErrorMsgFormat = "Could not convert value '%s' (String) for %s field '%s'";
```

Comment:
These are reversed. 
```suggestion
        String expectedErrorMsgFormat = "Could not convert value '%s' (String) for %s field '%s'";
        String expectedTimeErrorMsgFormat = "'%s’ is not a valid %s for %s using U.S. date parsing (MDY)";
```

Reply:
We actually do want to use the old format for Time (different from Date). I updated `expectedErrorMsgFormat` to `expectedDateErrorMsgFormat` to help clarify.